Papyrus/Sources/KeyMapping.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 /// Represents the mapping between your type's property names and
 /// their corresponding request field key.
-public enum KeyMapping {
+public enum KeyMapping: Sendable {
```

Comment:
Potentially dumb question but why does an `enum` need to be Sendable? Maybe I'm missing something but since enums have no properties to modify what's the possible race condition here?

Reply:
On public structs/enums, you have to explicitly mark the struct/enum Sendable. 
Without marking the enum Sendable, it could be possible that an associated value of some case in the enum is not Sendable which results in possible race conditions.

Reply:
Got it - thanks for the explanation!